transport/http_util.go

```diff
@@ -170,7 +170,7 @@ func newHPACKDecoder() *hpackDecoder {
 						return
 					}
 					// Extract the application user agent string.
```

Comment:
This is now incorrect. The application level can be setting multiple agent strings. Instead of splitting on space and taking the first element, this should look for the *last* space (strings.LastIndex), and take everything before that.